lib/velum/autoyast_values.rb

```diff
@@ -0,0 +1,47 @@
+# Load values that will be used to populate the autoyast profile
+class AutoyastValues
```

Comment:
let's wrap that into a `Velum` namespace like in all the other libs?

```ruby
module Velum
  class AutoyastValues
    ..
  end
end
```

Reply:
Forgot that, when moving the file - I added the `module`.